WordPress/src/main/res/values/strings.xml

```diff
@@ -256,6 +256,7 @@
     <string name="posts_draft_empty">You don\'t have any draft posts</string>
     <string name="posts_trashed_empty">You don\'t have any binned posts</string>
     <string name="multiple_status_label_delimiter" translatable="false">\u0020·\u0020</string>
+    <string name="post_list_toggle_item_view_layout">toggle rows view</string>
```

Comment:
Might be nitpicking, but to improve accessibility, should we describe the change instead: "Switch to cards view" and "Switch to list view"?

Reply:
Good point! I'll create another PR for the fix so we don't block the PRs which are dependant on this one. Even though it's a really small change, I'll need to create an UiState for menu item to keep things consistent with other VMs. Does it sounds good?

Reply:
Yep sure, time to cascade merge everything :)

Reply:
Thanks;)!

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9778